### PR TITLE
Increase image size module3

### PR DESCRIPTION
**Pull request by flor14**

### CODE REVIEW

**Comment by flor14** [comment]
@joelostblom this is what I have already recorded if you want to have a look.

File: .ipynb_checkpoints/Untitled-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (joelostblom) [inline .ipynb_checkpoints/Untitled-checkpoint.ipynb:1]: Remove this checkpoint file before merging

File: Untitled.ipynb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (joelostblom) [inline Untitled.ipynb:1]: Remove this file before merging

File: chapters/en/slides/module3/module3_04.md [hunk 3]
```diff
@@ -183,10 +196,12 @@ You have now tracked this change and made it part of the history of your project
 
 <center>
 
-<img src="/module3/vc-commit-editor.png" width="600px" alt="404 image" >
+<img src="/module3/vc-commit-editor.png" width="800px" alt="404 image" >
 
 </center>
 
+`git commit`
```
Comment (joelostblom) [inline chapters/en/slides/module3/module3_04.md:203]: I think it might look better if this is above the image

**Review by joelostblom — CHANGES_REQUESTED** [review]
Looks great overall! Just a couple of temporary files to remove before merging

File: Untitled.ipynb (hunk 2 above)
Comment (flor14) [reply Untitled.ipynb:1]: Thank you, I had not set my `.gitignore_global` file in the new computer yet

**Pull request merged**